_data-prepper/pipelines/configuration/processors/add-entries.md

```diff
@@ -18,6 +18,10 @@ You can configure the `add_entries` processor with the following options.
 | :--- | :--- | :--- |
 | `entries` | Yes | A list of entries to add to an event. |
 | `key` | Yes | The key of the new entry to be added. Some examples of keys include `my_key`, `myKey`, and `object/sub_Key`. |
+| `metadata_key` | Yes | The key of the new metadata attribute to be added. Argument must be a literal string key and not JsonPointer. One of key or metadata_key is required. |
+| `format` | No | A format string to use as value of the new entry to be added. For example, ${key1}-${ke2} where key1 and key2 are existing keys in the event. Required if value is not specified. |
```

Comment:
What do you mean by, "Required if value is not specified?"

```suggestion
| `format` | No | A format string to use as the value of the new entry, for example, `${key1}-${key2}` where `key1` and `key2` are existing keys in the event. Required if a value is not specified. |
```

Reply:
`format` Required if `value` and `value_expression` is not specified

Reply:
Backticks are not required for key1 and key2. Required if a value is not specified means `format` option is required If `value` and `value_expression` are not specified.